doc/060_forget.rst

```diff
@@ -213,12 +213,66 @@ All snapshots are evaluated against all matching ``--keep-*`` counts. A
+   0a1f9759  2019-09-01 11:00:00  mopped                  /tmp/foo
+   46cfe4d5  2019-09-08 11:00:00  mopped                  /tmp/foo
+   f6b1f037  2019-09-15 11:00:00  mopped                  /tmp/foo
+   eb430a5d  2019-09-22 11:00:00  mopped                  /tmp/foo
+   8cf1cb9a  2019-09-29 11:00:00  mopped                  /tmp/foo
+   5d33b116  2019-10-06 11:00:00  mopped                  /tmp/foo
+   b9553125  2019-10-13 11:00:00  mopped                  /tmp/foo
+   e1a7b58b  2019-10-20 11:00:00  mopped                  /tmp/foo
+   8f8018c0  2019-10-27 11:00:00  mopped                  /tmp/foo
+   59403279  2019-11-03 11:00:00  mopped                  /tmp/foo
+   dfee9fb4  2019-11-10 11:00:00  mopped                  /tmp/foo
+   e1ae2f40  2019-11-17 11:00:00  mopped                  /tmp/foo
+   ---------------------------------------------------------------
+   12 snapshots
+
+Then ``forget --keep-daily 4`` will keep the last four snapshots for the last
+four Sundays, but remove the rest:
+
+.. code-block:: console
+
+   $ restic forget --keep-daily 4 --dry-run
+   repository f00c6e2a opened successfully, password is correct
+   Applying Policy: keep the last 4 daily snapshots
+   keep 4 snapshots:
+   ID        Time                 Host        Tags        Reasons         Paths
+   -------------------------------------------------------------------------------
+   8f8018c0  2019-10-27 11:00:00  mopped                  daily snapshot  /tmp/foo
+   59403279  2019-11-03 11:00:00  mopped                  daily snapshot  /tmp/foo
+   dfee9fb4  2019-11-10 11:00:00  mopped                  daily snapshot  /tmp/foo
+   e1ae2f40  2019-11-17 11:00:00  mopped                  daily snapshot  /tmp/foo
+   -------------------------------------------------------------------------------
+   4 snapshots
+
+   remove 8 snapshots:
+   ID        Time                 Host        Tags        Paths
+   ---------------------------------------------------------------
+   0a1f9759  2019-09-01 11:00:00  mopped                  /tmp/foo
+   46cfe4d5  2019-09-08 11:00:00  mopped                  /tmp/foo
+   f6b1f037  2019-09-15 11:00:00  mopped                  /tmp/foo
+   eb430a5d  2019-09-22 11:00:00  mopped                  /tmp/foo
+   8cf1cb9a  2019-09-29 11:00:00  mopped                  /tmp/foo
+   5d33b116  2019-10-06 11:00:00  mopped                  /tmp/foo
+   b9553125  2019-10-13 11:00:00  mopped                  /tmp/foo
+   e1a7b58b  2019-10-20 11:00:00  mopped                  /tmp/foo
+   ---------------------------------------------------------------
+   8 snapshots
+
+The result of the ``forget --keep-daily`` operation does not depend on when it
+is run, it will only count the days for which a snasphot exists. This is a
+safety feature: it prevents restic from removing snapshots when no new ones are
+created. If it was implemented otherwise, running ``forget --keep-daily 4`` on
```

Comment:
`s/If it was implemented otherwise/Otherwise/` or `s/If it was implemented otherwise/If it was implemented differently/`, but the former is probably better.